examples/msgpack/src/tests.rs

```diff
@@ -0,0 +1,48 @@
+extern crate rmp_serde;
```

Comment:
I don't believe we can decode the response from the `msgpack_get` test, on line 26 of tests.rs, without this crate, and that whole test is rather pointless without that ability.